### DISCUSSION TITLE
Vimdoc and conceallevel isssues ?

### DISCUSSION THREAD

**Discussion by adam-devel** [neovim/neovim#31481] [Q&A]
This is what i see when i run neovim 0.10.2 with the flags `-u NORC` and `+h` to open the help documentation:
![screenshot](https://github.com/user-attachments/assets/573f06fa-c60f-421a-a725-63c7f992aa40)

Notice the "e.g." in the first line followed by nothing.
The default conceallevel is "2", i checked.

---
This is how the help docs look after `set conceallevel=1`:
![screenshot](https://github.com/user-attachments/assets/4ddeb5f3-5bdb-4cbe-b368-0523b32ad737)
Notice the empty space after "e.g.". Although it's hidden, if i set my cursor on the empty space and press "CTRL-]" it takes me to a section named "Bars example", which is the intended behavior.

---
Setting conceallevel to 0 has the following expected results:
![screenshot](https://github.com/user-attachments/assets/74ba557b-6211-4c4e-9023-2723c119a600)

---
is it intended behavior that links are concealed ? the `:h api` page is unusable with that conceal level.
If this behavior isn't intended, which i believe it isn't, how do i go about troubleshooting it ?

**Comment by zeertzjq** [comment]
How did you install Nvim 0.10.2? Did the `vimdoc` (or `help`) treesitter parser get installed along with it?

**Reply by adam-devel** [reply]
I installed the Arch Linux package of neovim through pacman: `pacman -S neovim`

Maybe this is a packaging issue ? if so I would expect more people to notice the issue

**Reply by clason** [reply]
You need to test with `nvim --clean`.

**Reply by adam-devel** [reply]
> Did the vimdoc (or help) treesitter parser get installed along with it?

It did get installed as a dependency. the package installs it as a shared library (`/usr/lib/libtreesitter.so`, `/usr/lib/libtreesitter.so.3` and `/usr/lib/libtreesitter.so.3.0`)
the version maybe relevant: the installed `tree-sitter-vimdoc` package version is 3.0.0

**Reply by adam-devel** [reply]
> You need to test with `nvim --clean`.

The issue got fixed !
What component could be the culprit ?

**Reply by clason** [reply]
Your config/plugins.

**Reply by adam-devel** [reply]
> Your config/plugins.

I don't even have a `~/.config/nvim/` folder currently. I deleted it a while ago to start from scratch

**Reply by clason** [reply]
Plugins will install in other places (which, depends on the plugin manager).

**Comment by clason** [comment]
Please test the provided appimages (with `--clean`, NOT `-u NORC`!).
